src/main.ts

```diff
@@ -1,3 +1 @@
-export {
-    ReactiveMembrane as default,
-} from './reactive-membrane';
+export { ReactiveMembrane as default } from './reactive-membrane';
```

Comment:
While doing some research for this PR, I found out that Webpack is recommending to [favor named exports over default exports](https://webpack.js.org/guides/package-exports/#guidelines) to avoid discrepancies between tooling. 

In general, we also favor named exports in all our packages except this one. We have an opportunity to correct this with the next major version. What do you think?